etc/config/snowball.amazon.properties

```diff
@@ -0,0 +1,26 @@
+# Version names: s[version integer]
+# Version example: 1.3.4 -> s010304
+
+compilers=&snowball
+objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
+defaultCompiler=s000007
+
+# TODO:
+# demangler=
+
+#buildenvsetup=ceconan
+#buildenvsetup.host=https://conan.compiler-explorer.com
+
+group.snowball.compilers=s000007
```

Comment:
This seems like a pretty tricky default ID. It's fine, and once it's set we won't be able to change it, so as long as you're OK with the number of `0`s and all that, this is fine :)

Reply:
These IDs are global and appear in URLs, can we pick something more obviously related to snowball?

Reply:
I added a comment explaining how it works the id versions

Reply:
> These IDs are global and appear in URLs, can we pick something more obviously related to snowball?

I didn't read this, sorry... Let me change it